src/kOS/Function/Suffixed.cs

```diff
@@ -405,4 +407,52 @@ public override void Execute(SharedObjects shared)
             shared.Cpu.PushStack(new WaypointValue(point, shared));
         }
     }    
+
+    [Function("transferall")]
+    public class FunctionTransferAll : FunctionBase
+    {
+        public override void Execute(SharedObjects shared)
+        {
+            var transferTo = shared.Cpu.PopValue();
+            var transferFrom = shared.Cpu.PopValue();
+            var resourceName = shared.Cpu.PopValue().ToString();
```

Comment:
The above three shared.Cpu.PopValue()'s  should be replaced with PopValueAssert(shared), after user_functions is merged in.